js/contextMenu.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
```

Comment:
also you could create a subdir for the code: js/menus and move both this and menuTemplate.js into that dir.